### ISSUE TITLE
Twitter keys ENV variables

### ISSUE THREAD

**Issue by andreicek** [notwaldorf/tiny-care-terminal#6]
The ones that are currently in use look terribly generic and I can see them causing issues in the future. Would it make sense to make them scoped like all the configuration ones?

Eg.
```
CONSUMER_KEY='...'
CONSUMER_SECRET='...'
ACCESS_TOKEN='...'
ACCESS_TOKEN_SECRET='...'
```

would be
```
TTC_CONSUMER_KEY='...'
TTC_CONSUMER_SECRET='...'
TTC_ACCESS_TOKEN='...'
TTC_ACCESS_TOKEN_SECRET='...'
```

**Comment by gasparch** [comment]
Without looking further - storing any secure information in ENV variables is bad. They are visible with 'ps' command to every other user on the computer. 

Secure way is to store them in the file and pass file name in ENV variable.

**Comment by hiendv** [comment]
@gasparch It seems ridiculous. [See](https://security.stackexchange.com/questions/14000/environment-variable-accessibility-in-linux/14009#14009). Any PoC? E.g.
```bash
ps eww [pid]
```

**Comment by gasparch** [comment]
I need to re-check my beliefs about ENV on different OSes. Back in time it was insecure way of storing info.

Also Linux is not only Unix/Unixlike OS to exist in the world :)))

**Comment by hiendv** [comment]
@gasparch Well, seeing that you mentioned `ps` actually triggered me

> Also Linux is not only Unix/Unixlike OS to exist in the world :)))

Agree. I would like to have some PoCs under other OSs. But I doubt that would be the responsibility of OSs. Not applications. Disclaimer: I'm a fan of [The Twelve-Factor App](https://12factor.net/config)

**Comment by gasparch** [comment]
I did not managed to test it on FreeBSD, recent Ubuntu/CentOS seems not have this problem anymore.

Anyway, env variables are terrible - they leak everywhere - cronjobs, child shellscripts, etc. Also you cannot change them without restarting process. but anyway, this is out of scope of this ticket.

**Comment by notwaldorf** [comment]
@andreicek i had originally kind of left them vague like that in case you set twitter keys for a different project. im worried if i change them now, people's stuff will break :(

**Comment by hiendv** [comment]
@notwaldorf Tag a new version maybe?

**Comment by Robophil** [comment]
after installing this from npm, how do i set the env variables ?

**Comment by notwaldorf** [comment]
@Robophil I've added some instructions here: https://github.com/notwaldorf/tiny-care-terminal/blob/master/README.md#setting-the-environment-variables

It really depends on what you're using :(

**Comment by notwaldorf** [comment]
Fixed in b8ead55

**Comment by Robophil** [comment]
Thanks @notwaldorf I'll check it out
